src/write-checksum.ts

```diff
@@ -0,0 +1,44 @@
+import fs from 'fs';
+import slugify from 'slugify';
+import { StorePlugin, calculateChecksum, getPluginsData } from './utils';
+import { format } from 'prettier';
+
+async function main(name?: string) {
+  const plugins: StorePlugin[] = await getPluginsData();
+  
+  if (name) {
+    const plugin = plugins.find(
+      (plugin) =>
+        slugify(plugin.name, { lower: true }) === slugify(name, { lower: true })
```

Comment:
Are we sure we'll add a whooooole lib to to that!? A simple regex would not be enough?

Reply:
hmm I am using slugify because I am using this script with the cli
How would you use the regex ?

Reply:
Im not sure I am following you. What are you looking for when use slugify? Turn a name-of-something in name_of_something!?

Reply:
If I want to be able to find `Massa Wallet` when I am running `npm run massa-wallet`

Reply:
No, I mean in this line:

```slugify(plugin.name, { lower: true }) === slugify(name, { lower: true })```

Are you trying to transform `Massa Wallet` into `massa-wallet` to be able to compare them!? Right!?

If yes, you definitely don't need a whooooole lib to do that!

Reply:
```ts
function slugify (str: string): string {
  return str.trim().toLowerCase().replace(/\s+/g, '-');
}

console.log(slugify('Something Else'));
// something-else
```

Reply:
I see, thanks!